src/web-ui/src/public/Footer.vue

```diff
@@ -1,6 +1,19 @@
 <template>
   <div class="footer">
     <div class="container">
+      <div class="row" v-if="user">
+        <form class="mt-2 mb-2 mx-auto text-center">
+          <div class="form-group mb-2 mr-sm-2">
+            <label for="sms-alerts">Join Text Alerts and get 20% off</label>
```

Comment:
Just curious: How do you check if Pinpoint is enbaled before showing the text alerts?